guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>Can we confirm that <code>Hosted Service</code>  shows up in typeKeywords of feature services, regardless of versions of portal/arcgis online? If so, is it better to put down some descriptions of how "hosted service" in the keywords distinguish hosted FS from feature server extensions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<ul><li>I talked to the author of the clone_items function and the PM for ArcGIS Online, who both believe the <code>typeKeywords</code>  property is a reliable method for filtering our hosted features services. The other way would be to query the <code>serviceDefinition</code>   for the word 'Hosted'. Using the <code>item</code> is much simpler and straightforward. There may be edge cases if you add a hosted feature layer as an item or <em>Save as a Copy</em>. </li></ul>